[IMP] theme_cobalt: revamp the theme

Brieuc-brd opened the pull request:
task-4178067
Part of task-4177978

| Before | After |
|--------|--------|
|  ![image](https://github.com/user-attachments/assets/a8a046a7-f0d0-4051-8904-f0e0c111799a) | ![cobalt_screenshot](https://github.com/user-attachments/assets/26ff1884-f595-47bf-8f56-0f606373f011) |

robodoo commented:
[![Pull request status dashboard](https://mergebot.odoo.com/odoo/design-themes/pull/936.png)](https://mergebot.odoo.com/odoo/design-themes/pull/936)

loco-odoo commented on theme_cobalt/views/customizations.xml:
```diff
@@ -5,7 +5,11 @@
+        <attribute name="data-oe-shape-data">{'shape':'web_editor/Origins/04_001','flip':['y'],'showOnMobile':false,'shapeAnimationSpeed':'0'}</attribute>
+    </xpath>
+    <xpath expr="//div[hasclass('container')]" position="before">
+        <div class="o_we_shape o_web_editor_Origins_04_001" style="background-image: url('/web_editor/shape/web_editor/Origins/04_001.svg?c3=o-color-3&amp;flip=y');"/>
```
You should also probably modify the `new_page_template.xml` file :+1: 
Indeed, now, if you add a new page with a banner (e.g. The last one of the 'About' category), you have something like :
```.xml
<section>
     <div class="o_web_editor_Origins_11_001"/>
     <div class="o_web_editor_Origins_04_001"/>
<section/>
```

loco-odoo reviewed (changes requested):
Thanks for your PR @Brieuc-brd ! :pray: 
I made a few suggestions on it. Also, do not forget to rebase your PR as it seems to be in conflict :+1:

Brieuc-brd commented on theme_cobalt/static/src/scss/primary_variables.scss:
```diff
@@ -25,20 +29,28 @@ $o-theme-font-configs: (
     ),
 );
 
-$o-selected-color-palettes-names: append($o-selected-color-palettes-names, 'cobalt-1');
```
Because `default-8` is already in the palette list so it's not necessary to append it again 👍 
https://github.com/odoo/design-themes/pull/927#discussion_r1772984204

Brieuc-brd commented on theme_cobalt/views/customizations.xml:
```diff
@@ -526,7 +575,7 @@
     </xpath>
     <xpath expr="//h2/following-sibling::div" position="attributes">
-        <attribute name="style" add="font-size: 7.75rem;" remove="font-size: 10.75rem;" separator=";"/>
+        <attribute name="style" remove="font-size: 10.75rem;" separator=";"/>
```
There was an issue with this snippet customization indeed. But it seems @bso-odoo was quicker to solve it 😅 Thanks !
https://github.com/odoo/design-themes/commit/d28cc926c2ae2b9daf37e7904b0a91234e76dcfb#diff-c3c637a4180d06d9e41a62900c4298a25a3976386b2268ea98a25d5029700b7eL529

Brieuc-brd commented:
Thanks for your review @loco-odoo 🙏 
I fixed the issues and rebased the branch : 
- Diff 1 : https://github.com/odoo/design-themes/compare/e0f2576b7c14079c0db73698fcc7181af2274933..b290eba0403bc23282f8f58df8351ef07a5c84ec
- Rebase + conflict resolved : https://github.com/odoo/design-themes/compare/b290eba0403bc23282f8f58df8351ef07a5c84ec..1c967e51a2c3c9a25ad432a54979b511ac699e0b
- Diff 2 : https://github.com/odoo/design-themes/compare/1c967e51a2c3c9a25ad432a54979b511ac699e0b..76a2ccdceaea42916fefb6127170b889466f525a

Could you take a look ?
Thanks in advance !

bso-odoo commented:
@robodoo delegate=loco-odoo

loco-odoo commented:
Thanks for the changes @Brieuc-brd :pray: 
Maybe can you also modify the `new_page_template_landing_1_s_banner` for the same reason than for the `new_page_template_about_s_banner` one?
Otherwise, LGTM :+1:

Brieuc-brd commented:
Good catch @loco-odoo  ! 👍 
I've made the change : https://github.com/odoo/design-themes/compare/76a2ccdceaea42916fefb6127170b889466f525a..bf78223889830f0348657f8f2444ce323435d4e3
And then rebased the branch.

I let you do the final check :)

qsm-odoo commented:
Being moved to https://github.com/odoo/design-themes/pull/969 (hopefully, with correct Signed-off-by)
Don't push here or there anymore, if you need to apply fixes, create a new PR to merge once the main one is merged.
